content/pt-br/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -0,0 +1,125 @@
+
+### (3/5) Planeje o deployment do seu cluster
+
+O Kubespray oferece a habilidade de customizar vários aspectos do deployment:
+
+* Escolha do modo de criação: kubeadm ou non-kubeadm
+* Plugins CNI (rede)
+* Configuração de DNS
+* Escolha da camada de gerenciamento: nativo/binário ou conteinerizado
+* Versões de componentes
+* Refletores de rota Calico
+* Opções de agente de execução do contêiner
+  * {{< glossary_tooltip term_id="docker" >}}
+  * {{< glossary_tooltip term_id="containerd" >}}
+  * {{< glossary_tooltip term_id="cri-o" >}}
+* Métodos de geração de certificados
+
+As customizações do Kubespray podem ser feitas em um [arquivo de variáveis](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html). Se você estiver começando com o Kubespray, considere usar os padrões do Kubespray para implantar seu cluster e explorar o Kubernetes.
+
+### (4/5) Deploy um Cluster
+
+Em seguida, instale seu cluster:
+
+Instalação do cluster usando [ansible-playbook](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#starting-custom-deployment).
+
+```shell
+ansible-playbook -i your/inventory/inventory.ini cluster.yml -b -v \
+  --private-key=~/.ssh/private_key
+```
+
+Grandes instalações (mais de 100 nós) podem exigir [ajustes específicos](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/large-deployments.md) para obter melhores resultados.
+
+### (5/5) Verifique a instalação
+
+O Kubespray fornece uma maneira de verificar a conectividade entre pods e a resolução de DNS com o [Netchecker](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/netcheck.md). O Netchecker garante que os pods de agentes do netchecker possam resolver solicitações de DNS e pingar cada um dentro do namespace padrão. Esses pods imitam um comportamento semelhante ao restante das cargas de trabalho e servem como indicadores de integridade do cluster.
+
+## Operações do Cluster
+
+O Kubespray fornece playbooks adicionais para gerenciar seu cluster: _dimensionar_ e _atualizar_.
+
+### Dimensione seu cluster
+
+Você pode adicionar nós de processamento de seu cluster executando o playbook de escalonamento. Para obter mais informações, consulte "[Adicionando nós](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#adding-nodes)".
+Você pode remover nós de processamento de seu cluster executando o playbook remove-nó. Para obter mais informações, consulte "[Remove nodes](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/getting-started.md#remove-nodes)".
```

Comment:
I think there was a bit of confusion here. The link text can stay as "removendo nós" (as opposed to "adicionando nós", since this is a title in the target document). But "remove-node" should not be translated because it is a playbook name (see https://github.com/kubernetes-sigs/kubespray/blob/6f1352eb533692a3ac4fe3561fb6b99572fd519d/docs/getting-started.md?plain=1#L67).